src/Controller/Component/RequestHandlerComponent.php

```diff
@@ -121,7 +122,7 @@ public function implementedEvents()
      * @param \Cake\Http\Response $response The response instance.
      * @return void
      */
-    protected function _setExtension($request, $response)
+    protected function _setExtension(\Cake\Http\ServerRequest $request, \Cake\Http\Response $response): void
```

Comment:
You can use my FQCN to use statement sniff if you dont want to do that manually :)